.goreleaser.yaml

```diff
@@ -23,3 +24,26 @@ builds:
     tags:
       - netgo
       - muslc
+  - id: linux-arm64
+    main: ./cmd/archwayd
+    binary: archwayd
+    hooks:
+      pre:
+        - wget https://github.com/CosmWasm/wasmvm/releases/download/v1.1.1/libwasmvm_muslc.aarch64.a -O /usr/local/lib/libwasmvm_muslc.a
```

Comment:
I'm wondering if this could be inherited from somewhere rather than being hard-coded in such a way that we don't need to remember that if we upgrade wasmvm in code we need to touch it here too, would this be possible somehow?